Apps/Medication/src/Services/RestMedicationStatementService.cs

```diff
@@ -74,6 +74,8 @@ public async Task<HNMessage<List<MedicationStatement>>> GetMedicationStatements(
             HNMessage<List<MedicationStatement>> hnClientMedicationResult = await this.RetrieveMedicationStatements(hdid, protectiveWord).ConfigureAwait(true);
             if (hnClientMedicationResult.Result == HealthGateway.Common.Constants.ResultType.Sucess)
             {
+                // Filter the results to return only Dispensed or Filled prescriptions.
+                hnClientMedicationResult.Message = hnClientMedicationResult.Message.Where(rx => rx.PrescriptionStatus == 'D' || rx.PrescriptionStatus == 'F').ToList<MedicationStatement>();
```

Comment:
Instead of strings, lets make them constants

Reply:
Resolved